Tasks/PowerShellOnTargetMachines/PowerShellOnTargetMachines.ps1

```diff
@@ -1,175 +1,177 @@
-param (
-    [string]$environmentName,    
-    [string]$adminUserName,
-    [string]$adminPassword,
-    [string]$protocol,
-    [string]$testCertificate,
-    [string]$resourceFilteringMethod,
-    [string]$machineNames,
-    [string]$scriptPath,
-    [string]$scriptArguments,
-    [string]$initializationScriptPath,
-    [string]$runPowershellInParallel,
-    [string]$sessionVariables
-    )
-
-Write-Verbose "Entering script PowerShellOnTargetMachines.ps1" 
-Write-Verbose "environmentName = $environmentName" 
-Write-Verbose "adminUserName = $adminUserName" 
-Write-Verbose "protocol = $protocol" 
-Write-Verbose "testCertificate = $testCertificate" 
-Write-Verbose "resourceFilteringMethod = $resourceFilteringMethod" 
-Write-Verbose "machineNames = $machineNames" 
-Write-Verbose "scriptPath = $scriptPath" 
-Write-Verbose "scriptArguments = $scriptArguments"
-Write-Verbose "initializationScriptPath = $initializationScriptPath"
-Write-Verbose "runPowershellInParallel = $runPowershellInParallel"
-Write-Verbose "sessionVariables = $sessionVariables"
+[CmdletBinding()]
+param()
+
+Trace-VstsEnteringInvocation $MyInvocation
+
+# Get inputs for the task
+$environmentName = Get-VstsInput -Name EnvironmentName
+$adminUserName = Get-VstsInput -Name AdminUserName
```

Comment:
Should this be a required input?

Reply:
Should you also use this rev of task to validate if the user input is in proper format?